test/lib/httpReceiver.test.js

```diff
@@ -391,7 +392,7 @@ describe('http receiver', () => {
   it('should send a response', (done) => {
     function taskReceivedCallback(receivedTask, callback) {
       receivedTask.should.be.an.Object();
-      receivedTask.taskType.should.eql('dataLink');
+      receivedTask.taskType.should.eql('data');
```

Comment:
I/we haven't changed any MAPI or KCS taskType strings. Since this is the test for `httpReceiver`, is it guaranteed that `data`/`functions` (in this context) won't break any other parts of the code?